package.json

```diff
@@ -74,7 +74,8 @@
   "devDependencies": {
     "ava": "^0.19.1",
     "jest": "^20.0.4",
-    "nyc": "^10.0.0"
+    "nyc": "^10.0.0",
+    "rss": "^1.2.2"
```

Comment:
Thanks for the clarification, I just assumed that it would pull in all packages whether dev or not.  That is good to know!